SUPPORT.md

```diff
@@ -0,0 +1,34 @@
+## How do I&hellip;?
```

Comment:
Stands for **h**orizontal **ellip**sis&hellip; brother of `&vellip;` the vertical ellipsis

&vellip;

In case you were wondering :)